src/improvements/tasks/sep/004-opcm-upgrade-v300/VALIDATION.md

```diff
@@ -0,0 +1,457 @@
+```
+
+The resulting calldata sent from the ProxyAdminOwner safe is thus:
+
+```
+0x82ad56cb000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000020000000000000000000000000fbceed4de885645fbded164910e10f52febfab35000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000164ff2dd5a100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000003000000000000000000000000034edd2a225f7f429a63e0f1d2084b9e0a93b538000000000000000000000000189abaaaa82dfc015a588a7dbad6f13b1d3485bc03ee2917da962ec266b091f4b62121dc9682bb0db534633707325339f99ee4050000000000000000000000004ca9608fef202216bc21d543798ec854539baad3000000000000000000000000ff9d236641962cebf9dbfb54e7b8e91f99f10db003ee2917da962ec266b091f4b62121dc9682bb0db534633707325339f99ee40500000000000000000000000005c993e60179f28bf649a2bb5b00b5f4283bd525000000000000000000000000d7db319a49362b2328cf417a934300cccb442c8d03ee2917da962ec266b091f4b62121dc9682bb0db534633707325339f99ee40500000000000000000000000000000000000000000000000000000000
+```
+
+<!-- TODO(#740) - Add a link to the gov post -->
+In mainnet runbooks, this calldata should appear in [Action Plan]() section of the Governance proposal.
+
+# State Changes
+
+## Nested Safe State Overrides and Changes
+
+<!--TODO(#740) - I see additional overrides to the ProxyAdminSafe not documented in NESTED-VALIDATION.md -->
+This task is executed by the nested 2/2 `ProxyAdminOwner` Safe. Refer to the
+[generic nested Safe execution validation document](../../../../../NESTED-VALIDATION.md)
+for the expected state overrides and changes.
+
+<!--TODO(#740) - Figure out how to retrieve the $SAFE_HASH, I don't see it printed in the output -->
+The `approvedHashes` mapping of the `ProxyAdminOwner` should change during the simulation.
+See the ["Key Computation" section](../../../../../NESTED-VALIDATION.md#key-computation) in the nested validation doc
+for instruction on how to validate this change.  The `$SAFE_HASH` value for this calculation should be:
+- `0x7ae8800316ad25e257c77b54d034e5befc61dc38da97801125cc99f404be529e`
```

Comment:
@blmalone - wondering if you can help me sort out the 2 todos here?

Reply:
Responded about the first below. 

For the second TODO, I believe it would be this value in the output: 
![CleanShot 2025-03-26 at 14 14 39@2x](https://github.com/user-attachments/assets/c749b951-af57-4b3b-a247-39a88d163efc)

Reply:
Thanks - I just recently rebased on `main`, and I'm seeing this in the output now 🎉